snippets/javascript.json

```diff
@@ -140,9 +127,9 @@
     "prefix": "hpreg",
     "description": "Hapi.js Load One Plugin",
     "body": [
-      "server.register(require('${plugin}'), (err) => {",
+      "await server.register(require('${plugin}'), (err)=>{",
```

Comment:
I think extra space must be preserved, it is more common as in https://github.com/deerawan/vscode-hapijs-snippets/pull/2/files#diff-d068deda6bfd39601979b8a531b379b1R141